testutil/kubernetes_helper.go

```diff
@@ -294,3 +294,12 @@ func (h *KubernetesHelper) URLFor(namespace, deployName string, remotePort int)
 
 	return pf.URLFor(""), nil
 }
+
+// ChangeK8sCfgContextNamespace modifies the current KubeConfig context
+// to use the provided namespace as the default for all subsequent kubectl commands
+func (h *KubernetesHelper) ChangeK8sCfgContextNamespace(namespace string) (string, error) {
```

Comment:
Can't you leverage `func (h *KubernetesHelper) Kubectl(stdin string, arg ...string)` somehow ?

Reply:
Done!